src/pages/login/index.tsx

```diff
@@ -84,15 +87,23 @@ function LoginPage() {
                 className="!mb-1 mt-3"
                 onChange={handleChange('email')}
               />
-              <Input
-                type="password"
-                value={values.password}
-                error={touched.password ? errors.password : undefined}
-                required
-                placeholder={t('login:password')}
-                className="!my-1"
-                onChange={handleChange('password')}
-              />
+              <div className="relative">
+                <Input
+                  type={showPassword ? 'text' : 'password'}
+                  value={values.password}
+                  error={touched.password ? errors.password : undefined}
+                  required
+                  placeholder={t('login:password')}
+                  className="!my-1"
+                  onChange={handleChange('password')}
+                />
+                <span
+                  className="absolute right-3 top-2 cursor-pointer"
+                  onClick={() => setShowPassword(!showPassword)}
+                >
+                  {showPassword ? '\u{1F440}' : '\u{1F648}'}
+                </span>
+              </div>
```

Comment:
Whole implementation should be placed in `Input` component for password type. Thanks to this, in the future we will be able to make changes in only one place

Reply:
> ```
>           <div className="relative">
>             <Input
>               type={showPassword ? 'text' : 'password'}
>               value={values.password}
>               error={touched.password ? errors.password : undefined}
>               required
>               placeholder={t('login:password')}
>               className="!my-1"
>               onChange={handleChange('password')}
>             />
>             {showPassword ? (
>               <EyeIcon
>                 onClick={() => setShowPassword(false)}
>                 className="absolute right-4 top-4 h-6 w-6 text-gray-400 cursor-pointer"
>               />
>             ) : (
>               <EyeOffIcon
>                 onClick={() => setShowPassword(true)}
>                 className="absolute right-4 top-4 h-6 w-6 text-gray-400 cursor-pointer"
>               />
>             )}
>           </div>
> ```
> 
> @Ryczko can we go like this?

Still we need to put it into `Input` component